Formula/e/enex2notion.rb

```diff
@@ -21,7 +21,7 @@ class Enex2notion < Formula
 
   depends_on "certifi"
   depends_on "pymupdf"
-  depends_on "python@3.12"
+  depends_on "python@3.13"
```

Comment:
In addition to failure, needs to be done with `pymupdf`.

Reply:
make sense, but looks like need llvm to move to py3.13 for mupdf  as well.

Reply:
`llvm` can be done in separate PR (though will probably take at least a day to build) since the Clang bindings are created for all Pythons (3.8 onward) and the runtime dependency is for LLDB.